include/swift/SwiftRemoteMirror/SwiftRemoteMirrorTypes.h

```diff
@@ -71,6 +82,17 @@ typedef struct swift_reflection_info {
   swift_reflection_ptr_t RemoteStartAddress;
 } swift_reflection_info_t;
 
+/// Represents the set of Swift reflection sections of an image,
+/// but doesn't assume that the sections share contiguous memory.
```

Comment:
There is no real guarantee that reflection is contiguous.  I think that it is sufficient to say:

```
Represents the Swift reflection metadata from an image.
```

Reply:
I wanted to emphasize that this data structure doesn't assume the memory isn't contiguous to contrast the already existing `swift_reflection_info`, which _does_ assume that. Should I remove that?

Reply:
I think that the clarification that was made further up is _critical_ here.  It isn't that the reflection info is non-contiguous.  The metadata layout _is_ contiguous.  The mapping from local to remote is non-contiguous.  Is that understanding incorrect?  If not, this is misleading, and I would say that this is better named as perhaps `swift_remote_reflection_data_slice` to indicate that this is a sliver of the full metadata information.